tests/manage/z_cluster/test_add_capacity_flexible_scaling.py

```diff
@@ -0,0 +1,92 @@
+import logging
+
+from ocs_ci.framework import config
+from ocs_ci.framework.pytest_customization.marks import (
+    skipif_bm,
+)
+from ocs_ci.framework.testlib import (
+    ignore_leftovers,
+    ManageTest,
+    tier1,
+    acceptance,
+    flexible_scaling_required,
+)
+from ocs_ci.utility.localstorage import check_pvs_created
+from ocs_ci.ocs.node import add_disk_to_node
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.ocp import OCP
+from ocs_ci.ocs.resources.pod import get_osd_pods
+from ocs_ci.ocs.cluster import check_ceph_health_after_add_capacity
+from ocs_ci.ocs.resources.storage_cluster import (
+    osd_encryption_verification,
+    get_storage_cluster,
+)
+from ocs_ci.framework.pytest_customization.marks import skipif_managed_service
+from ocs_ci.ocs.ui.helpers_ui import ui_add_capacity_conditions
+from ocs_ci.ocs.node import get_nodes
+from ocs_ci.ocs.ui.base_ui import login_ui, close_browser
+from ocs_ci.ocs.ui.add_replace_device_ui import AddReplaceDeviceUI
+
+
+logger = logging.getLogger(__name__)
+
+
+@ignore_leftovers
+@tier1
+@acceptance
+@skipif_bm
+@flexible_scaling_required
+@skipif_managed_service
+class TestAddCapacityFlexibleScaling(ManageTest):
+    """
+    Automates adding variable capacity to the flexible scaling cluster
+    """
+
+    def test_add_capacity_flexible_scaling(self):
+        """
+        Test to add variable capacity to the OSD cluster
+
+        """
+        existing_osd_pods = get_osd_pods()
+        node_objs = get_nodes(node_type=constants.WORKER_MACHINE)
+        add_disk_to_node(node_objs[0])
```

Comment:
I am not sure if this test covers the scenario for https://bugzilla.redhat.com/show_bug.cgi?id=1943280?
Shouldn't we add more than 1 additional disk for OSDs and then try to expand to see if all disks are consumed for the scaling?

Reply:
Please make sure you cover below,
a) Have >2 disks for OSDs on a node and do add-capacity. 
b) Note that the way the disks are consumed for OSD is different from UI and CLI
Lets say you have Y available disks on a node and  X as the initial count.
1) When add-capacity is performed from UI, all the available disks will be used for OSDs as we will not have the control of the disks count.
2) Incase of Add capacity using YAML, we can specify how many OSDs can be used.
e.g. Lets say we have 5 Available disks, we can still use only 4 by changing count to X+4 (rest 1 PVs stay in Available state and can be used for future).  Please make sure to cover these scenarios with proper validations

Reply:
Done, We add 2 disks to the same node. 
In this example the OSD size is 0.5T.
http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/jnk-pr6270b3548/jnk-pr6270b3548_20220913T085353/logs/ui_logs_dir_1663063509/screenshots_ui/test_add_capacity_lso/2022-09-13T10-11-22.173670.png